tests/admin_scripts/tests.py

```diff
@@ -236,6 +236,13 @@ def test_builtin_with_bad_environment(self):
         self.assertNoOutput(out)
         self.assertOutput(err, "No module named '?bad_settings'?", regex=True)
 
+    def test_default_module_settings_environment(self):
+        "invalid env var: django-admin builtin commands succeed if environment variable is wrongly set"
```

Comment:
suggestion: "commands that don't need settings succeed if settings file doesn't exist"